### ISSUE TITLE
`require-jsdoc`: exempt unbracketed arrow functions with no arguments and literal return value

### ISSUE THREAD

**Issue by brettz9** [gajus/eslint-plugin-jsdoc#746]
## Motivation

Avoid need for extra documentation.

## Current behavior

This is reported:

```js
const arrow = () => 55.3
```

## Desired behavior

For this kind of very simple (and compiler-detectable) method, add an option to avoid need for documentation.

## Alternatives considered

Suffer. :-)

**Comment by brettz9** [comment]
On second thought, I think we should perhaps just encourage `contexts` to be set to `ArrowFunctionExpression[expression] > *:not(Literal):not(Identifier[name="undefined"])`
